assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -77,6 +79,8 @@ export const Minischedule = ({
   view: "run" | "block"
 }) => {
   const [showPast, setShowPast] = useState<boolean>(false)
+  const routes = useRoutes()
```

Comment:
This will make a new api call. Since the routes are already in the properties panel, could they be passed in as an argument?